likecoin/js/admin/likecoin_metabox.js

```diff
@@ -36,55 +64,212 @@ async function onISCNCallback(event) {
   if (event.origin !== 'https://like.co') {
     return;
   }
+  lcPostInfo.mainStatusTitle = 'READY to #DePub';
+  lcPostInfo.arweaveAndISCNUploadStatus = 'uploading to ISCN...';
```

Comment:
How can we handle i18n of this/these string?

Reply:
pass in from php as discussed